server/cromwell.py

```diff
@@ -30,6 +29,17 @@
     get_cromwell_key,
 )
 
+CROMWELL_REQUIRED_PARAMS = [
```

Comment:
This seems a bit brittle in the sense that we might easily forget to extend this list when adding a parameter.

Previously, we caught the exception that's raised when an element isn't present in the dictionary. I completely agree that the resulting error message wasn't great. However, would there be a way to retrieve the name of the missing key from the exception to create a better error message, making this explicit list redundant?

Reply:
Yep, it does this as well, I've just moved that error message creation to the generic exception handling bit: 
https://github.com/populationgenomics/analysis-runner/blob/90de4c6493e6c4970439185a67de41085f333c95/server/main.py#L167-L171

I'm happy to remove these required_param checks though, I agree with the brittleness.

Reply:
I've removed the required_param checks.

Reply:
Cool idea to use a middleware for this!